runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -251,6 +315,13 @@ J9::AheadOfTimeCompile::dumpRelocationData()
 
       cursor++;
 
+      uint8_t *newCursor = self()->dumpRelocationHeaderData(origCursor, isVerbose);
```

Comment:
another spot for a simple comment, please, to explain why the code is structured in this way.

Reply:
Heh I added the comment https://github.com/eclipse/openj9/pull/3324/files#diff-424266007eb87d9b2ac057d002798798R209 so that it was consistent with the previous review comment about the switch statements (since I understood the review as requesting comments for the switch statements only); I'll add a comment for the calls to `dumpRelocationHeaderData` and `initializeCommonAOTRelocationHeader`

Reply:
Comments added